iquip/apps/builder.py

```diff
@@ -253,6 +302,8 @@ def run(self):
         except TypeError:
             print("Failed to convert the build arguments to a JSON string.")
             return
+        for schedOptName, schedOptValue in self.schedOpts.items():
+            params[schedOptName] = schedOptValue
```

Comment:
You can use [`dict.update()`](https://docs.python.org/3/library/stdtypes.html#dict.update): `params.update(self.schedOpts)`